### ISSUE TITLE
[Research] OUI Compliance audit of the `saved_objects` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4092]
Tracking issue for OUI compliance audit of the `saved_objects` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
56 | [src/plugins/saved_objects/public/save_modal/saved_object_save_modal.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/saved_objects/public/save_modal/saved_object_save_modal.scss) | 
37 | [src/plugins/saved_objects/public/save_modal/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/saved_objects/public/save_modal/_index.scss) | 
30 | [src/plugins/saved_objects/public/index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/saved_objects/public/index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by sayuree** [comment]
@BSFishy Hi, I want to work on this issue.

**Comment by sayuree** [comment]
> @BSFishy Hi, I want to work on this issue.

@BSFishy I finished audit of the saved_objects plugin: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4161
